<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TomahawkWindow</class>
 <widget class="QMainWindow" name="TomahawkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tomahawk</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="hline1">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>1</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QWidget" name="sidebarWidget" native="true"/>
      <widget class="QWidget" name="playlistWidget" native="true"/>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="hline2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>1</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionToggleConnect">
   <property name="text">
    <string>Go &amp;Online</string>
   </property>
  </action>
  <action name="actionAddFriendManually">
   <property name="text">
    <string>Add &amp;Friend...</string>
   </property>
  </action>
  <action name="actionUpdateCollection">
   <property name="text">
    <string>U&amp;pdate Collection</string>
   </property>
   <property name="toolTip">
    <string>Update Collection</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Configure Tomahawk...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionLoadXSPF">
   <property name="text">
    <string>Load &amp;XSPF...</string>
   </property>
  </action>
  <action name="actionCreatePlaylist">
   <property name="text">
    <string>Create &amp;New Playlist...</string>
   </property>
  </action>
  <action name="actionAboutTomahawk">
   <property name="text">
    <string>About &amp;Tomahawk...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionCreateAutomaticPlaylist">
   <property name="text">
    <string>Create New &amp;Automatic Playlist</string>
   </property>
  </action>
  <action name="actionCreate_New_Station">
   <property name="text">
    <string>Create New &amp;Station</string>
   </property>
  </action>
  <action name="actionShowOfflineSources">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Offline Sources</string>
   </property>
  </action>
  <action name="actionHideOfflineSources">
   <property name="text">
    <string>Hide Offline Sources</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+Z</string>
   </property>
  </action>
  <action name="actionLegalInfo">
   <property name="text">
    <string>&amp;Legal Info...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionDiagnostics">
   <property name="text">
    <string>Diagnostics...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionRescanCollection">
   <property name="text">
    <string>Fully &amp;Rescan Collection</string>
   </property>
   <property name="toolTip">
    <string>Fully Rescan Collection</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_btnResolve_clicked()</slot>
 </slots>
</ui>
